src/mm-dialog/index.html

```diff
@@ -0,0 +1,142 @@
+<!DOCTYPE html>
+<html>
+	<head>
+		<script language="javascript" src="../../bower_components/webcomponentsjs/webcomponents-lite.js"></script>
+		<script type="text/javascript" src="//cdnjs.cloudflare.com/ajax/libs/jquery/1.11.1/jquery.js"></script>
+		<link rel="import" href="../mm-button/mm-button.html">
+		<link rel="import" href="../mm-header/mm-header.html">
+		<link rel="import" href="mm-dialog.html">
+		<style type="text/css">
+			body, html {
+				height: 100%;
+				min-height: 100%;
+			}
+
+			body {  
+				margin:0;
+				padding:0;
+				background: #eee;
+			}
+
+			.col {
+				-webkit-box-sizing: border-box;
+				-moz-box-sizing: border-box;
+				box-sizing: border-box;
+				display: block;
+				float: left;
+				padding: 0 20px;
+				/*font-size: 0;*/
+			}
+
+			.col.c0 {
+				width:400px;
+			}
+
+			.col.c1 {
+				width:300px;
+			}
+
+			.col.c2 {
+				width:200px;
+			}
+
+			.col.c3 {
+				width:500px;
+			}
+
+			.col.c4 {
+				width:50%;
+			}
+
+			.col:after {
+				visibility: hidden;
+				display: block;
+				/*font-size: 0;*/
+				content: " ";
+				clear: both;
+				height: 0;
+			}
+
+			p {
+				font-family: sans-serif;
+				color: #333;
+				font-size: 12px;
+				line-height: 18px;
+				margin: 20px 0;
+			}
+
+			.bold { font-weight: bold; }
+
+			hr {
+				border: 0;
+				display: block;
+					border-bottom: 1px solid #fff;
+					border-top: 1px solid #ccc;
+					height: 0px;
+					margin-top: 20px;
+			}
+
+			.vr {
+				width:0px;
+				height: 100%;
+				border-right: 1px solid #fff;
+				border-left: 1px solid #ccc;
+				display: block;
+				float: left;
+			}
+
+			.cbc {
+				margin-top: 10px;
+			}
+		</style>
+	</head>
+	<body>
+		<div class="col c4">
+			<mm-dialog id="testDialog"
+				type="info"
+				header="It works"
```

Comment:
We need to handle a potentially multiline header. Currently, the text floats beneath the icon when the line breaks. I'd probably a) inline the icon, so text breaks beneath it or b) put a block next to the icon and allow the text to break to the right of the space the icon occupies.